Dev4Agriculture.ISO11783.ISOXML/Analysis/ISODeviceAnalysis.cs

```diff
@@ -1,5 +1,7 @@
-﻿using System.Collections.Generic;
+﻿using System;
+using System.Collections.Generic;
 using System.Linq;
+using de.dev4Agriculture.ISOXML.DDI;
 using Dev4Agriculture.ISO11783.ISOXML.IdHandling;
```

Comment:
Remove unrequired usings please ;)

Reply:
<img width="321" alt="image" src="https://github.com/dev4Agriculture/isoxml-dotnet/assets/127326880/0049b716-6d30-4ac4-8308-0b6edecb025b">
It's require to use DDIList